### PR TITLE
[#1] Set up Rails Template

### LINKED ISSUE #1: [Chore] Set Up the Project with Rails Template
## Why

Kickstart the project with Rails Template.
## Acceptance Criteria

Project can run locally.

### PR DESCRIPTION
**Pull request by blyscuit**
- Close #1

## What happened 👀

Generate the project with [Rails-Template](https://github.com/nimblehq/rails-templates). with API only `bundle exec rails new google-search-ruby -m https://raw.githubusercontent.com/nimblehq/rails-templates/develop/template.rb --api`.

## Insight 📝

- Uses Addons:
  - docker
  - GitHub
  - Devise
- Add CODEONWERS

## Proof Of Work 📹

Screen Shot 2023-05-29 at 16 51 35

### CODE REVIEW

**Comment by malparty** [comment]
Should we use the `--api` option from the Rails Template? That might save you some time/effort to clean everything later 💭 

I've just tried, the best way is to download the Rails template repository in your laptop, use the `develop` branch (as the last fix was not released yet 🙇 My bad), and run `rails new new-api -m ../rails-templates/template.rb --api` ;-) 

Then move in this folder, run `bundle` just to be sure, and you should be able to run `make dev` ;-) 


This API variant won't have assets (JS/stylesheets/...) folders, so less to clean later: 
image

File: .github/wiki/Home.md [hunk 1]
```diff
@@ -0,0 +1 @@
+> *Insert information about your project here!*
```
Comment (malparty) [inline .github/wiki/Home.md]: Should we replace this line with a the name of the application and what it does?

**Review by malparty — APPROVED** [review]
Approved with a minor suggestion! 🚀

**Review by byhbt — APPROVED** [review]

File: .github/wiki/Home.md (hunk 1 above)
Comment (blyscuit) [reply .github/wiki/Home.md]: Updated 🙏

**Pull request merged**